cogs/api.py

```diff
@@ -1,13 +1,8 @@
 import shutil
 
-import discord, datetime, glob, json, os, random, requests, string, sys
+import datetime, glob, json, os, random, requests, string
 from pathlib import Path
-from discord.ext import commands
-
-if not os.path.isfile("config.py"):
-    sys.exit("'config.py' not found! Please add it and try again.")
-else:
-    import config
```

Comment:
This should already error when starting the `bot.py` so I don't see a reason to have error handling here.